internals/webhooks/subnamespace_webhook.go

```diff
@@ -271,6 +283,24 @@ func ValidateCreateSnsRequest(sns *utils.ObjectContext, parentQuotaObj *utils.Ob
 	return true, ""
 }
 
+// ValidateAllResourceQuotaParamsValid validates that all quota params: storage, cpu, memory, gpu exists and are positive
+// if one of the params is not valid, the function returns false with the relevant error
+func ValidateAllResourceQuotaParamsValid(sns *utils.ObjectContext) (bool, error) {
+	quotaSNS := utils.GetSnsQuotaSpec(sns.Object).Hard
+	ResourceQuotaParams := danav1.ZeroedQuota.Hard
+	for res := range ResourceQuotaParams {
+		requestedRes := quotaSNS[res]
+		if requestedRes.Format == "" {
+			return false, errors.New(denyMessageWithoutResourceRequest + res.String())
```

Comment:
I think the error message should include all the quota params that must be included. If a user tried to create a subnamespace with just cpu for example, he'd potentially face an error 3 more times before including all the quota parameters